packages/link/src/Link.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+
+type Props = {
+  /** Content of the link. Can be anything */
+  children: MnrlReactNode,
+  /** Link renders an <a> by default but you can specify a different tag or component as the root (e.g. a React Router Link).*/
+  root?: MnrlReactNode
+};
+
+/**
+ * Link component
+ */
+export default function Link({ children, root = 'a', ...restProps }: Props) {
+  if (restProps.onClick) {
```

Comment:
I like the intention of enforcing the distinction between buttons and link, but I think logging an error or even a warning can potentially disrupt developer workflows, and we would never want to do this in production.  I think that we'll need to solve this via documentation once we begin writing usage guidelines.